### DISCUSSION TITLE
Opendrift configuration with current and wind files (with different spatial resolution)

### DISCUSSION THREAD

**Discussion by esahuc** [OpenDrift/opendrift#787] [Q&A]
Hello all, 

I was wondering if it is possible to create an Oceandrift configuration (with winddrift activated) where the current data have a 4km resolution and wind data have a 1.3km resolution ? Do the code will handle the difference of resolution and adapt to the coarser resolution ?

thanks in advance,

Best regards

**Comment by knutfrode** [comment]
Yes, combining readers with different resolutions and projections is
possible.
See the gallery for several examples.

**Reply by esahuc** [reply]
Thank you for your help, i'm now trying a drift example simply with my MARC2D data from Ifremer, but facing the following issue. Is there any link with the U and V variable of my file not having the same name as the one in the example ?

Thanks,
![Opendrift_error](https://user-images.githubusercontent.com/95405500/144403079-26c12200-c08d-4a80-89f3-7d70e0f1b418.PNG)

**Comment by knutfrode** [comment]
I see that you have version 1.5.6, and quite a lot happened until the recent version (1.7.3), including a fix for the bug that you encounter:
https://opendrift.github.io/history_link.html

So I would recomment updating OpenDrift and its dependencies (or re-installing from scratch):
https://opendrift.github.io/install.html
